src/Octoshift/Services/GithubApi.cs

```diff
@@ -1142,4 +1142,37 @@ private static Mannequin BuildMannequin(JToken mannequin)
             StartColumn = (int)scanningAlertInstance["location"]["start_column"],
             EndColumn = (int)scanningAlertInstance["location"]["end_column"]
         };
+
+    public virtual async Task<bool> AbortMigration(string migrationSourceId)
+    {
+        var url = $"{_apiUrl}/graphql";
+
+        var query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        var gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    }
+                ) {
+                    success
+                    }
```

Comment:
it looks like this curly brace is extra

Reply:
hahah yes got there eventually! also double quotes around the variable in the JSON blob